src/gui/guiFormSpecMenu.cpp

```diff
@@ -284,12 +284,13 @@ void GUIFormSpecMenu::parseSize(parserData* data, const std::string &element)
 		data->invsize.Y = MYMAX(0, stof(parts[1]));
 
 		lockSize(false);
+#ifndef __ANDROID__
 		if (parts.size() == 3) {
 			if (parts[2] == "true") {
 				lockSize(true,v2u32(800,600));
```

Comment:
800x600 was our previous default desktop screensize, does this need updating to the new default of 1024x600?

Reply:
If changed, spaces after the commas if possible.

Reply:
> does this need updating to the new default of 1024x600?

I guess it probably should be for desktop but I am not sure changing that is this PR is a good idea. I am not even entirely sure of its purpose and there may be a risk of it effecting the look of some menus/forms. As you can see, this section is simply ignored for android.